AGENTS.md

```diff
@@ -0,0 +1,197 @@
+# AGENTS.md
+
+## Setup
+
+### Requirements
+
+- Node.js >=20.0.0
+- Python 3
+- Git (for development)
+
+### Environment
+
+- Docker (required for E2E tests via CHT Docker Helper)
+- Internet connection (for downloading dependencies and connecting to CHT instances)
```

Comment:
Minor, but I don't _think_ we make any automatic connections to remote CHT instances as a part of the automated build/test process.

```suggestion
- Internet connection (for downloading dependencies)
```